samples/hosting/aws-lambda-http-sqs/SQSLambda_1/AwsLambda.Messages/SQSTriggerMessage.cs

```diff
@@ -0,0 +1,5 @@
+﻿using NServiceBus;
+
+public class SQSTriggerMessage : IMessage
```

Comment:
I think I see the intent of why this was renamed, however, all the other messages are still being handled by SQS and they don't have the SQS "prefix" in their name. Although this is a sample and is not production code and this sample is very specific to SQS. Have we considered the implications of showing a practice of marrying the message type names to the underlying transport?